tests/Application/src/Entity/OrderItem.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
Is this how it should look that we create an entity for tests?

Reply:
Plugin requires base Sylius model extending, so I think yes.